korim/src/commonMain/kotlin/com/soywiz/korim/bitmap/BitmapSlice.kt

```diff
@@ -94,31 +81,22 @@ interface BmpCoordsWithT<T : ISizeInt> : BmpCoords, Closeable, Resourceable<BmpC
         val bw = width.toFloat()
         val bh = height.toFloat()
 
-        if (isRotatedInBaseDeg90) {
-            // Base is rotated
-            val xDim = if (imageOrientation.isRotatedDeg90CwOrCcw) w else h
-            val yDim = if (imageOrientation.isRotatedDeg90CwOrCcw) h else w
-            tlX = tl_x + y / bh * dx
-            tlY = tl_y + x / bw * dy
-            brX = tlX + xDim / bh * dx
-            brY = tlY + yDim / bw * dy
-            trX = tlX
-            trY = brY
-            blX = brX
-            blY = tlY
-        } else {
-            // Base is not rotated
-            val xDim = if (imageOrientation.isRotatedDeg90CwOrCcw) h else w
-            val yDim = if (imageOrientation.isRotatedDeg90CwOrCcw) w else h
-            tlX = tl_x + x / bw * dx
-            tlY = tl_y + y / bh * dy
-            brX = tlX + xDim / bw * dx
-            brY = tlY + yDim / bh * dy
-            trX = brX
-            trY = tlY
-            blX = tlX
-            blY = brY
-        }
+        val flipDim = isRotatedInBaseDeg90 xor ioRotated
```

Comment:
```suggestion
        val flipDim = isRotatedInBaseDeg90 xor isRotated
```

Reply:
io should stand for imageOrientation, but I have no Problem to apply this change

Reply:
Okay. Then lets keep as it is now. Ill trry to review it in detail later today and merge it if possible 👍

Reply:
Currently working on it 😄